<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StoryWidget</class>
 <widget class="QDialog" name="StoryWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Стори</string>
  </property>
  <layout class="QVBoxLayout" name="StoryWidgetVerticalLayout" stretch="0,0,0,0,2">
   <item>
    <widget class="QLabel" name="nameLabel">
     <property name="font">
      <font>
       <family>GOST type B</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Name</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="epicGroupBox">
     <property name="font">
      <font>
       <family>GOST type B</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Эпик</string>
     </property>
     <layout class="QHBoxLayout" name="epicGroupBoxHorizontalLayout">
      <item>
       <widget class="QLabel" name="epicLabel">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Epic</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignJustify|Qt::AlignVCenter</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0">
     <item>
      <widget class="QGroupBox" name="developerGroupBox">
       <property name="font">
        <font>
         <family>GOST type B</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="title">
        <string>Разработчик</string>
       </property>
       <layout class="QHBoxLayout" name="developerGroupBoxHorizontalLayout">
        <item>
         <widget class="QLabel" name="developerLabel">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Developer</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="storyPointsGroupBox">
       <property name="font">
        <font>
         <family>GOST type B</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="title">
        <string>S. pts</string>
       </property>
       <layout class="QHBoxLayout" name="storyPointsGroupBoxHorizontalLayout">
        <item>
         <widget class="QLabel" name="storyPointsLabel">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>10</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QComboBox" name="stateComboBox">
     <property name="font">
      <font>
       <family>GOST type B</family>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="tasksTableWidget">
     <property name="font">
      <font>
       <family>GOST type B</family>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
